lib/paper_trail/model_config.rb

```diff
@@ -168,6 +168,31 @@ def cannot_record_after_destroy?
         ::ActiveRecord::Base.belongs_to_required_by_default
     end
 
+    def type_aware_has_many(klass)
```

Comment:
In keeping with the name of the method `setup_associations`, I would call this `setup_versions_association`.

Reply:
Good suggestion!